src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -117,7 +117,7 @@ const AppContent = React.memo(({ repo, repoVersions, onRequestUpgrade }) => {
             >
               <DetailsGroup compact={compact}>
                 <Heading2>Description</Heading2>
-                <div>{description}</div>
+                <div>{description || 'No description.'}</div>
```

Comment:
Description has a [default value of empty string](https://github.com/aragon/aragon/blob/master/src/apps/AppCenter/InstalledApps/AppContent.js#L27), would it make sense to update it to 'No description'?

Reply:
Fixed in 72f52c0.